VERSION

```diff
@@ -1 +1 @@
-2.1.1.pre
+2.1.2.pre
```

Comment:
This represents official gitlab shell version. I think we should not change that value. We do not to reference this value anywhere - deployment is done using git branch.